karma.conf.js

```diff
@@ -52,7 +52,7 @@ module.exports = function(config) {
 		colors: true,
 		logLevel: config.LOG_INFO,
 
-		browsers: ['PhantomJS2'],
+		browsers: ['Chrome'],
```

Comment:
You're absolutely right ! I find phantomjs less intrusive too.
But with angular2 we absolutely need to use `phantomjs2` (at least it's what i understood !).

I had troubles to setup `phantomjs2`, that's why i changed to chrome.

How do you want me to do that ? Let `chrome` and add a comment saying that  it's possible to use `PhantomJS2` instead (or both) ?